lib/sorbet-rails/model_plugins/active_record_attribute.rb

```diff
@@ -12,8 +12,9 @@ class ColumnType < T::Struct
     sig { returns(String) }
     def to_s
       type = base_type.to_s
-      type = "T.nilable(#{type})" if nilable
+      # A nullable array column should be T.nilable(T::Array[column_type]) not T::Array[T.nilable(column_type)]
       type = "T::Array[#{type}]" if array_type
+      type = "T.nilable(#{type})" if nilable
```

Comment:
thanks for catching this bug

Reply:
Eh..... I wrote it 😄

Reply:
😅 Well I'm glad you come back and fix it